### PR TITLE
Add Mount Sinai as an industrial user.

### PR DESCRIPTION
**Pull request by hammer**
Hey @smondet want to confirm the description of Ketrew looks good?

### CODE REVIEW

File: site/learn/companies.md [hunk 1]
```diff
@@ -320,6 +320,17 @@
 </dl>
 <dl class="row">
     <dt class="span3">
+        <img src="/img/users/mount-sinai.gif" alt="Mount Sinai">
+    </dt>
+    <dd class="span5">
+        <h2><a href="http://www.mountsinai.org">Mount Sinai</a>, United States</h2>
+        <p>The <a href="http://www.hammerlab.org">Hammer Lab</a> at Mount Sinai develops and uses <a href="https://github.com/hammerlab/ketrew">Ketrew</a>
+        for managing complex bioinformatics workflows. Ketrew includes an embedded domain-specific language to simplify the specification of complex workflows and
```
Comment (Chris00) [inline site/learn/companies.md]: Could you wrap lines at 80 chars.  Thanks.

**Comment by Chris00** [comment]
Are you sure you want that much white space around the logo?  Why not crop it?

File: site/learn/companies.md (hunk 1 above)
Comment (hammer) [inline site/learn/companies.md]: I could, but none of the other descriptions have been wrapped at 80 chars. What is the motivation for starting now?

Comment (Chris00) [inline site/learn/companies.md]: On Sat, 24 Jan 2015 01:18:13 -0800, Jeff Hammerbacher wrote:

> In site/learn/companies.md:
> 
> > @@ -320,6 +320,17 @@
> >  </dl>
> >  <dl class="row">
> >      <dt class="span3">
> > -        Mount Sinai
> > -    </dt>
> > -    <dd class="span5">
> > -        <h2><a href="http://www.mountsinai.org">Mount Sinai</a>, United States</h2>
> > -        <p>The <a href="http://www.hammerlab.org">Hammer Lab</a> at Mount Sinai develops and uses <a href="https://github.com/hammerlab/ketrew">Ketrew</a>
> > -        for managing complex bioinformatics workflows. Ketrew includes an embedded domain-specific language to simplify the specification of complex workflows and
> 
> I could, but none of the other descriptions have been wrapped at 80 chars. What is the motivation for starting now?

Oh, I did not pay attention.  I meant it more like a general rule.  This won't hinder merging it anyway.

**Comment by hammer** [comment]
I can't really evaluate the aesthetics of the logo as the site doesn't build locally for me (`make deps` fails with `No package matches opam2web>1.3.1`, oddly). I chose an image that was around the same dimensions as the images that I saw already up there. I can crop if you think it looks better.

**Comment by Chris00** [comment]
On Sat, 24 Jan 2015 01:30:18 -0800, Jeff Hammerbacher wrote:

> I can't really evaluate the aesthetics of the logo as the site
> doesn't build locally for me (make deps fails with No package
> matches opam2web>1.3.1, oddly).

Are you using opam 1.2?  Indeed

$ opam show opam2web --raw
opam-version: "1.2"
…

**Comment by hammer** [comment]
Yep using opam 1.2.0. Is the failure to build a known bug?

In any case, I just committed a cropped version of the logo. Let me know which you prefer.

**Comment by Chris00** [comment]
> Is the failure to build a known bug?

Not that I know.  What is returned by `opam show opam2web` (after an `opam update`).

**Comment by Chris00** [comment]
> I just committed a cropped version

You may want to squash your commits — we do not need to record the old version of the logo in the history.

**Comment by hammer** [comment]
Commits have been squashed, merge away.

Also, the build wasn't working because I had not done `opam update` recently. Sorry about that, I sometimes miss the obvious.

**Pull request merged**